src/WorkItemMigrator/tests/Migration.Common.Tests/MigrationCommonTests.cs

```diff
@@ -21,21 +20,15 @@ public void Setup()
             _fixture.Customize(new AutoNSubstituteCustomization() { });
         }
 
-        [Test]
-        public void When_generating_user_map_Then_map_is_correct()
+        [TestCase("a@jira.com", "a@azdo.com")]
+        [TestCase("b@jira.com", "b@azdo.com")]
+        public void When_generating_user_map_Then_map_is_correct(string source, string target)
         {
             string[] userMapLines = { "a@jira.com=a@azdo.com", "b@jira.com=b@azdo.com" };
             Dictionary<string, string> generatedUserMap = UserMapper.ParseUserMappings(userMapLines);
 
```

Comment:
please wrap this into Assert.Multiple

Reply:
Fixed by [13d30d3](https://github.com/solidify/jira-azuredevops-migrator/pull/470/commits/13d30d38b0675b475b952ea164099ce9d99eb483)